lib/node_modules/@stdlib/stats/base/dists/triangular/cdf/benchmark/benchmark.js

```diff
@@ -34,17 +35,26 @@ bench( pkg, function benchmark( b ) {
 	var mode;
 	var min;
 	var max;
+	var len;
 	var x;
 	var y;
 	var i;
 
+	len = b.iterations;
```

Comment:
```suggestion
	len = 100;
```

This change is also consistent throughout this PR.